modules/wkt/src/lib/encode-wkt.ts

```diff
@@ -38,24 +40,24 @@ export default function encodeWKT(gj: {
   }
 
   function wrapParens(s: string) {
-    return '(' + s + ')';
+    return `(${s})`;
```

Comment:
Not specifically related to this PR but is there a performance hit to defining these helper functions within this function instead of in higher scope? If you had 100,000 geometries that you wanted to encode to WKT, and did `geometries.map(encodeWKT)`, then each of these helper functions would be defined 100,000 times, instead of once? 

It looks like maybe this isn't an issue for modern browsers? https://stackoverflow.com/a/10204449 cc @ibgreen ?